homeassistant/components/image_processing/opencv.py

```diff
@@ -7,22 +7,56 @@
 from datetime import timedelta
 import logging
 
+import requests
+
+import voluptuous as vol
+
 from homeassistant.core import split_entity_id
 from homeassistant.components.image_processing import (
-    ImageProcessingEntity, PLATFORM_SCHEMA)
-from homeassistant.components.opencv import (
-    ATTR_MATCHES, CLASSIFIER_GROUP_CONFIG, CONF_CLASSIFIER, CONF_ENTITY_ID,
-    CONF_NAME, process_image)
+    CONF_SOURCE, CONF_ENTITY_ID, CONF_NAME, PLATFORM_SCHEMA,
+    ImageProcessingEntity)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['opencv-python==3.2.0.6', 'numpy==1.12.0']
```

Comment:
I have no better idea - not a python dev - can you give me a snippet that you think would work? I just tried running 3.2.0.7 on my pi and it didn't work (as you said).

would this just be

```
if cv2 is None:
  # do some shit here
  cv2 = something
...
```
???